zproject/local_settings_template.py

```diff
@@ -148,39 +151,47 @@
 # The email gateway provides, for each stream, an email address that
```

Comment:
For me, this text was a bit hard to read because it assumes that I have a mental concept of an 'email gateway' that I don't know about yet. From a newbie setup perspective, some wording like this would work better for me:

"Zulip supports emailing messages to be posted to streams by creating an email address for each stream to receive messages at. The format for the email address is defined by the EMAIL_GATEWAY_PATTERN configuration item. This is called 'email mirroring' in Zulip. 

[aside: why is the word 'local' in "There are two ways to make use of local email mirroring"? Is there a non-local way of doing email mirroring?]

"There are two ways to configure email mirroring in Zulip: 
...